add profile_picture field to PortalProfile model

lumpy72006 opened the pull request:
feature #6 

Hi @Mariatta, I:

Added `profile_picture` to `PortalProfile` and updated `portal_account/forms.py` to support image uploads.

Added pillow to `requirements.txt`, as it's required for image handling.

Refactored the form to use `{% bootstrap_form form %}` instead of listing fields explicitly, which changed the field order.

Implemented `order_fields()` in `PortalProfileForm` to ensure fields appear in the correct order.

Mariatta commented:
Thank you! This is a great start, but you forgot to add the migrations file.

Could you also add the profile picture selection to the PortalProfile form?

lumpy72006 commented:
Sure. But I added the profile picture in PortalProflile form in this line:
`fields = ["pronouns", "coc_agreement", "profile_picture"]`

lumpy72006 commented:
I updated the form’s HTML to use `{% bootstrap_form form %}` instead of listing fields explicitly, which caused field order issues. To fix this, `order_fields()` was added in `PortalProfileForm` to ensure fields appear in the intended order while keeping the form dynamic. 
I added `enctype="multipart/form-data"` to the form tag (important for file uploads like the profile picture).
Please review 🙏🏾

Mariatta commented:
Looks great! Thank you!

You'll also need to add these lines to the portal/urls.py


```
from django.conf import settings
from django.conf.urls.static import static

if settings.DEBUG:
    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

lumpy72006 commented:
I've done that now. Please, is the merge conflict on my end? I don't remember adding those lines 🥲. Can you resolve?

Mariatta commented:
Thank you for starting this. I think it looks great. I'll keep this open for now because we still need to figure out how to store the media files in production.

lumpy72006 commented:
Ohkay 🤝

Mariatta commented:
Can you please update the branch and resolve conflicts? I have set up the project to store media files on Digital Ocean so we can now proceed with this change.

Mariatta commented:
Please address the CI failure.
Try running `make reformat` and `make check`.

lumpy72006 commented:
Thank you 🙌🙌🙌

Mariatta commented:
Thanks for the PR!

lumpy72006 commented:
> Thanks for the PR!

Thank you 🙌
